feature/lostandfound/src/main/java/in/koreatech/koin/feature/lostandfound/ui/detail/component/DetailContent.kt

```diff
@@ -0,0 +1,145 @@
+package `in`.koreatech.koin.feature.lostandfound.ui.detail.component
+
+import android.net.Uri
+import android.webkit.URLUtil
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.dp
+import coil.compose.SubcomposeAsyncImage
+import coil.request.ImageRequest
+import `in`.koreatech.koin.core.designsystem.theme.KoinTheme
+import `in`.koreatech.koin.feature.lostandfound.R
+
+@Composable
+fun DetailContent(
+    imageUris: List<Uri>?,
+    content: String,
+    isWriterAdmin: Boolean,
+    modifier: Modifier = Modifier
+) {
+    // Check image url is valid
+    val filteredImageUrls = imageUris?.filter { URLUtil.isValidUrl(it.toString()) }
```

Comment:
viewModel에서 로직을 처리하거나 무거운 연산인 경우 remember을 사용해주세요.
```suggestion
    val filteredImageUrls = remeber(imageUrls) { imageUris?.filter { URLUtil.isValidUrl(it.toString()) } }
```

Reply:
viewModel에서 state에 값을 넣을때 처리하도록 수정했습니다!